cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -82,17 +92,14 @@ spec:
               fi;
       # BEGIN_PROMETHEUS_TO_SD
       - name: prometheus-to-sd-exporter
-        image: gcr.io/google-containers/prometheus-to-sd:v0.1.3
+        image: gcr.io/google-containers/prometheus-to-sd:v0.2.1
         command:
           - /monitor
```

Comment:
Flags component, target-port and whitelisted-metrics are deprecated. Please use --source instead: https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/prometheus-to-sd/main.go#L68

Reply:
Will it still work? WDYT about leaving it as it is and following-up in a separate PR?